WebRoot/public/extuserprov.jsp

```diff
@@ -67,7 +67,11 @@
 				<table>
 					<tr>
 						<td><app:img id="ZLoginErrorIcon" altkey='ALT_ERROR' src="dwt/ImgCritical_32.png" /></td>
-						<td id="errorMessage"></td>
+						<td id="errorMessage">
+							<% if (request.getAttribute("errorCode") != null) { %>
+								<fmt:message bundle="${zhmsg}" key='${errorCode}'/>
```

Comment:
I just referred use of `zhMsg` bundle used in other places of same file: https://github.com/Zimbra/zm-web-client/blob/854a2da5caef46f5b3f643cdff58ed0163105857/WebRoot/public/extuserprov.jsp#L139

So to be consistent, I follow same. 

Let me know if you have more insights on this.

Thanks